send-nft/index.js

```diff
@@ -86,7 +86,10 @@ function getGasFee(count) {
 async function run() {
   try {
     const data = await readCsv('list.csv');
-    const signer = await DirectSecp256k1HdWallet.fromMnemonic(MNEMONIC, { prefix: 'like' });
+    if (!MNEMONIC && !PRIVATE_KEY) throw new Error('need MNEMONIC or PRIVATE_KEY');
```

Comment:
Should we also guard the case that both `MNEMONIC` and `PRIVATE_KEY` are provided?

Reply:
added warning